dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/event/EventsExportControllerTest.java

```diff
@@ -269,6 +296,76 @@ void getEventByIdWithDataValues() {
     assertHasMember(dataValue, "storedBy");
   }
 
+  @ParameterizedTest
+  @ValueSource(
+      strings = {
+        "in:red",
+        "sw:bl",
+        "ew:een",
+        "like:red",
+        "ilike:green",
+        "nilike:yellow",
+        "!null",
+        "ne:yellow"
+      })
+  void shouldReturnEventsForMultiTextDataValuesUsingSupportedOperators(String operator) {
```

Comment:
```suggestion
  void shouldReturnEventsForMultiTextDataValuesUsingSupportedOperators(String filter) {
```
This is the filter: operator + value

Reply:
resolved